src/antares/service/api_services/link_api.py

```diff
@@ -60,7 +60,9 @@ def create_link(
         """
         base_url = f"{self._base_url}/studies/{self.study_id}"
         # TODO: Currently, AntaresWeb does not have a specific endpoint for links. Once it will, we should change this logic.
-        area1_id, area2_id = sorted([area_from.id, area_to.id])
+
+        area1_id, area2_id = sorted([area_from, area_to])
```

Comment:
We'll have the issue everywhere in the code. It could be nice to do this sort inside the Link object inside the `__init__`

You can remove this everywhere else if you do this

Reply:
Removed